src/KBS.Infrastructure/IManager.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+namespace KBS.Infrastructure
+{
+    public interface IManager
+    {
+        string GetState();
```

Comment:
It would be nice if you add some xml-docs here to explain what these methods should do.